src/components/App/App.js

```diff
@@ -64,19 +73,21 @@ class App extends Component {
           }}/>
           <Route exact path="/" render={() =>  
             <MoviesContainer 
-            movies={this.state.movies} 
-            updateClickedMovie={this.updateClickedMovie}
+            movies={this.state.displayedMovies} 
```

Comment:
Good addition, making this render displayedMovies instead of movieLibrary. This way we are only ever rendering from one place.